apps/els_core/src/els_config.erl

```diff
@@ -55,7 +55,8 @@
                | elvis_config_path
                | indexing_enabled
                | bsp_enabled
-               | compiler_telemetry_enabled.
+               | compiler_telemetry_enabled
+               | refactorerl.
```

Comment:
Considering that someone else is trying a similar approach with Wrangler (at least so I've heard), I wonder if the config should be a bit more generic, as in:

```
external_refactoring_tool:
  name: refactorerl | wrangler
```

Or similar.

Reply:
The proposal looks good to me.